[CIVIS-3109] Remove "beta" status from python ITR documentation (#21581)
DataDog/documentation: content/en/intelligent_test_runner/setup/python.md

{{% ci-itr-activation-instructions %}}

## Running tests with the Intelligent Test Runner enabled

> The Intelligent Test Runner will be enabled when you run tests with the Datadog integration active. Simply run your tests as normal:

Reviewer: ```suggestion
The Intelligent Test Runner is enabled when you run tests with the Datadog integration active. Run your tests with the following command:
```

{{< tabs >}}

{{% tab "Pytest" %}}

{{< code-block lang="shell" >}}
DD_ENV=ci DD_SERVICE=my-python-app pytest --ddtrace
{{< /code-block >}}

{{% /tab %}}

{{% tab "Unittest" %}}

{{< code-block lang="shell" >}}
DD_ENV=ci DD_SERVICE=my-python-app ddtrace-run python -m unittest
{{< /code-block >}}

{{% /tab %}}

{{< /tabs >}}

### Temporarily disabling the Intelligent Test Runner

The Intelligent Test Runner can be disabled locally by setting the `DD_CIVISIBILITY_ITR_ENABLED` environment variable to `false` or `0`.

`DD_CIVISIBILITY_ITR_ENABLED` (Optional)
: Enable the Intelligent Test Runner coverage and test skipping features<br />

> **Default**: `(true)`

Reviewer: ```suggestion
**Default**: `(true)`

Run the following command to disable the Intelligent Test Runner:
```
